index.html

```diff
@@ -718,12 +718,15 @@ <h3>Validate Algorithm</h3>
         </li>
         <li>
 Dereference the `statusListCredential` URL, and ensure that all
-proofs verify successfully. If the dereference fails, or if any of the proof
-verifications fail, return a validation error.
+proofs verify successfully. If the dereference fails, raise a
+<a href="#STATUS_RETRIEVAL_ERROR">STATUS_RETRIEVAL_ERROR</a>. If any of the
+proof verifications fail, raise a
+<a href="#STATUS_VALIDATION_ERROR">STATUS_VALIDATION_ERROR</a>.
```

Comment:
Maybe this should be `STATUS_VERIFICATION_ERROR` instead? Same for each other use of this error; it seems they may just be about verifying the data is authentic and proper vs. business rules around accepting the status information.

Reply:
I think the status fails to validate.

What the verifier chooses to do about that failure is then a business rules question.

Reply:
Ok, I can support that mental model.

Reply:
I would just like to check that we have the following alternative processes covered:
1. the verifier asks the issuer for a status list at some previous date.time (e.g. last week)
2. the issuer responds with a status retrieval error because such a list is not available
3. the verifier then either terminates or asks for the current status list
Alternatively 
1a. the verifier asks the issuer for a status list at some previous date.time (e.g. last week)
3a the issuer  ignores the date.time on the query, and returns the current status list.

Reply:
> I would just like to check that we have the following alternative processes covered:
>
> 1. the verifier asks the issuer for a status list at some previous date.time (e.g. last week)
> 1. the issuer responds with a status retrieval error because such a list is not available
> 1. the verifier then either terminates or asks for the current status list

Step 3 feels very much like business logic to me.

> Alternatively
> 1a. the verifier asks the issuer for a status list at some previous date.time (e.g. last week)
> 2a. the issuer ignores the date.time on the query, and returns the current status list.

I do not think (2a) is viable *unless* the status list VC includes its validity datetime range, which must be included whether or not the issuer supports historical status lists ... and I think there ought to be an advisory message along those lines ("You asked for a historical status list. I don't provide those. Here's my current status list.") to ensure that the verifier knows they're not looking at the historical snapshot they requested.

Reply:
@dlongley your concerns have been fixed in 6e6d6b963e8f6c62fb5c1a2f792bbefc6c4d0b67.

@David-Chadwick I'll address your concern in PR #112 since it has to do w/ queries for historical information.